src/Service/MagicLinks/MagicLinkService.cs

```diff
@@ -64,6 +65,8 @@ private async Task EnforceQuotaAsync(MagicLinkTokenRequest request)
         }
     }
 
+    private static bool IsAdminConsole(AccountMetaInformation account) => AdminConsole.Contains(account.Tenant);
```

Comment:
Suggestion: simplify to avoid additional noise with the field and keep all related logic collocated:

```suggestion
    private static bool IsAdminConsole(AccountMetaInformation account) =>
        string.Equals(account.Tenant, "admin", StringComparison.OrdinalIgnoreCase) ||
        string.Equals(account.Tenant, "adminconsole", StringComparison.OrdinalIgnoreCase);
```